exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillDatabaseMetaDataImpl.java

```diff
@@ -36,6 +42,7 @@
  */
 class DrillDatabaseMetaDataImpl extends AvaticaDatabaseMetaData
```

Comment:
A feature to set on the session option ANSI_QUOTES via jdbc connection string is added in a new commit.
For example: `"jdbc:drill:zk=local;ansi_quotes=true"`